app/pages/lab/subject-set.cjsx

```diff
@@ -131,8 +131,10 @@ EditSubjectSetPage = React.createClass
   subjectLimitMessage: (project_subject_count, user) ->
     "The project has " + project_subject_count + " uploaded subjects. " +
     "You have uploaded " + user.uploaded_subjects_count + " subjects from an " +
-    "allowance of " + user.subject_limit + ". Your uploaded subject count is the tally of all subjects (including those deleted) that your account has uploaded through the project builder or Zooniverse API."
-
+    "allowance of " + user.subject_limit + ". Your uploaded subject count is the tally of all subjects " +
+    "(including those deleted) that your account has uploaded through the project builder or Zooniverse API." +
+    "Please <a href="https://www.zooniverse.org/about/contact"> contact us</a> to request changes to your allowance."
```

Comment:
copy looks good, but I think you'll need to change / escape the `"` chars on the href, e.g. 
`"Please <a href='https://www.zooniverse.org/about/contact'> contact us</a> to request changes to your allowance."`